server/apps/publish/formatters/aap_ipnews_formatter_test.py

```diff
@@ -139,3 +140,65 @@ def TestMultipleCategories(self):
                     codes = set(doc['selector_codes'].split(' '))
                     expected_codes = set('cxx 0fh axx az and pxx 0ah 0ir 0px 0hw pnd pxd cnd cxd 0nl axd'.split(' '))
                     self.assertSetEqual(codes, expected_codes)
+
+    def TestGeoBlock(self):
+        article = {
+            'source': 'AAP',
+            'anpa_category': [{'qcode': 'a'}],
+            'headline': 'This is a test headline',
+            'byline': 'joe',
+            'slugline': 'slugline',
+            'subject': [{'qcode': '04001005'}, {'qcode': '15011002'}],
+            'anpa_take_key': 'take_key',
+            'unique_id': '1',
+            'type': 'text',
+            'body_html': 'body',
+            'word_count': '1',
+            'priority': '1',
+            'task': {'desk': 1},
+            'urgency': 1,
+            'place': [{'qcode': 'VIC', 'name': 'VIC'}],
+            'targeted_for': [{'name': 'New South Wales', 'allow': False}, {'name': 'Victoria', 'allow': True}]
+        }
+
+        with self.app.app_context():
+            subscriber = self.app.data.find('subscribers', None, None)[0]
+
+            f = AAPIpNewsFormatter()
+            seq, doc = f.format(article, subscriber)[0]
+            codes = set(doc['selector_codes'].split(' '))
+            expected_codes_str = 'an5 an4 an7 an6 ax5 an3 ax6 ax7 0hw an8 0ir px6 ax4 ax3 ax8 px5 0ah 0px px3 az'
+            expected_codes_str += ' px8 0fh px7 px4 pn3 pn4 pn5 pn6 pn7 px0'
+            expected_codes = set(expected_codes_str.split(' '))
+            self.assertSetEqual(codes, expected_codes)
+
+    def TestGeoBlockNotTwoStates(self):
```

Comment:
For consistency, I would suggest renaming these two test methods to start with lowercase (`Test...` --> `test...`).